soup/manager.go

```diff
@@ -0,0 +1,58 @@
+package soup
+
+import (
+	"github.com/sjuls/soup-ranking/dbctx"
+)
+
+var (
+	unknownSoupName = "Unknown soup"
+)
+
+type (
+	// Manager incapsulates soup business logic and exposes methods to interact with soup data
+	Manager struct {
+		repo    dbctx.SoupRepository
+		scraper Scraper
+	}
+)
+
+// NewManager creates a new manager
```

Comment:
Dear Linter,

Will you please shut up.

Regards,
Everyone who ever made a New-function